cinema/models.py

```diff
@@ -60,6 +60,12 @@ class Meta:
     def __str__(self):
         return self.movie.title + " " + str(self.show_time)
 
+    @property
+    def get_tickets_available(self):
+        return (self.cinema_hall.rows
+                * self.cinema_hall.seats_in_row
+                - self.tickets.count())
```

Comment:
The `get_tickets_available` property should correctly calculate the number of available tickets for a movie session. Ensure that it accounts for the total capacity of the cinema hall minus the number of tickets already sold. The current implementation seems to do this, but please verify that the `tickets` related name is correctly set up and used.